beheer.md

```diff
@@ -153,18 +163,20 @@ Er is dus een bepaalde behoefte.
+In beide gevallen kent de Product Owner een prioriteit toe. Daarna wordt i.s.m. met de ontwikkelaars bepaald hoeveel tijd het kost een bug danwel feature te verwerken. In geval van een bug wordt tevens nog gekeken of deze backwards compatible opgelost kan worden of niet. Op basis daarvan en de beschikbare financiele ruimte kan de Product Owner nu de roadmap gaan aanpassen.
 Vooralsnog is daarbij uitgangspunt dat er 2 x per jaar een nieuwe versie van de API standaard wordt gepubliceerd.
 Als de roadmap gereed is wordt eigenlijk ook pas duidelijk of het bij de geplande versies gaat om een nieuwe PATCH, MINOR of MAJOR.  
 
->Indien volgens de roadmap op 1 moment in het jaar zowel backwards compatible bugs als features in de API standaard moeten worden verwerkt worden eerst de bugs opgelost in de huidige versie (wat leidt tot een PATCH op die versie). Op basis van die versie wordt dan weer een MAJOR/MINOR vervaardigd. (_<-- Misschien opnemen in het versie-beheer verhaal._)
+Indien volgens de roadmap op 1 moment in het jaar zowel backwards compatible bugs als features in de API standaard moeten worden verwerkt worden eerst de bugs opgelost in de huidige versie (wat leidt tot een PATCH op die versie). Op basis van die versie wordt dan weer een MAJOR/MINOR vervaardigd. (_<-- Misschien opnemen in het versie-beheer verhaal._)
 Dit voorkomt dat een leverancier verplicht wordt de nieuwe functionaliteit te implementeren terwijl voor hem op dat moment wellicht alleen het oplossen van de bug van belang is.
 
 _Op dit moment is versiebeheer zo omschreven dat zowel een bug als een feature incompatible en compatible kan zijn. Een bug kan nu opgelost worden met een PATCH maar ook met een MAJOR. In dat laatste geval kun je heel goed zien aan het versienummer dat het niet compatible is met een voorgaande versie. Dat voelt voor mij echter erg vreemd. Het is een geheel andere definitie van MAJOR dan gebruikelijk in de SW wereld._
 
-
-
-_Hoe gaan we om met pull-requests van derden? Stel dat we die willen honoreren, nemen we die 1 op 1 over, gaan we die reengineeren of doen we nog iets anders?_
+Wanneer tezamen met een issue een pullrequest (PR) wordt ingediend wordt niet alleen bevinding of wens maar ook een uitwerking ingediend. Dan is het van belang naast de inhoud van het issue ook te kijken naar:
+* Past de PR in de roadmap? Was dit al iets wat voorzien was maar naar voren is gehaald vanuit de praktijk?
+* Is de PR een breaking change, met andere woorden, past de wijziging in het bestaande schema van releases?
+* Wat is de urgentie van de wijziging? Meestal zal een PR een zekere urgentie hebben omdat het een oplossing is voor iets wat bij een gemeente opgelost moest worden wat niet kon wachten tot een volgende release van de standaard.
```

Comment:
Dit is eigenlijk onderdeel van het bepalen van de prioriteit. Dit is iets dat ik een eindje terug al behandel bij 'In beide gevallen kent de Product Owner een prioriteit toe ...'. In die alinea kun je wat mij betreft aangeven dat urgentie een van de criteria is bij het bepalen van de prioriteit. Deze alinea moet hier n.m.m. echter weg.

Reply:
verwijderd